sjb/config/test_cases/test_branch_openshift_ansible_logging_json_file.yml

```diff
@@ -1,5 +1,11 @@
 ---
-parent: 'common/test_cases/origin_built_release.yml'
+parent: 'common/test_cases/origin_built_installed_release.yml'
+overrides:
+  sync_repos:
```

Comment:
i would expect this to be the same as in the other one -- an override on `sync:`

Reply:
outstanding